### ISSUE TITLE
Rename Master Branch To L

### ISSUE THREAD

**Issue by phelixbtc** [namecoin/namecoin-legacy#98]
...to legacy or master_legacy. I don't see any advantage to it any more and it is confusing that the master branch is completely outdated.

**Comment by domob1812** [comment]
Sounds good.  Actually, if you aren't opposed to major renamings, I suggest to do this:

master -> legacy
namecoinq -> master
namecoinq-rc -> master-rc

I did a wrong local merge already countless times due to "master" not being the actual master branch.

**Comment by phelixbtc** [comment]
I agree that your suggestion would be the obvious naming scheme. But to prevent any confusion I will only rename the master for now and the others in a couple of months.
